next.config.js

```diff
@@ -5,11 +5,23 @@ const nextConfig = {
   },
   images: {
     remotePatterns: [
+      //production image host
       {
         protocol: 'https',
         hostname: 'undfcbmldjkujposixvn\\.supabase\\.co',
         pathname: '/storage/v1/object/public/images/**',
       },
+      //local development image host
+      {
+        protocol: 'http',
+        hostname: '127.0.0.1',
+        pathname: '/storage/v1/object/public/images/**',
+      },
+      {
+        protocol: 'http',
+        hostname: 'localhost',
+        pathname: '/storage/v1/object/public/images/**',
+      },
```

Comment:
I'm actually a little lost on this one. The pathname doesn't really match my local bucket but this works 😅 are you sure we need both of these declarations though, wouldn't '127.0.0.1' have us covered?

Reply:
I know 😅

I think '127.0.0.1' should be enough - I was overly cautious and included both of them 🤸‍♂️ . It is now removed :)

In my local supabase new items pictures are stored at the following path, which is just the same as mentioned in the document:
![image](https://github.com/user-attachments/assets/0d04a137-fad2-4d2a-aa6b-8d7400c5b6ab)
@camelPhonso Just to double check if it is the same for both of us - where are your images stored when you open items/ profile tables?

Reply:
Just checked, this is the same for me yes 👍